include/gatekeeper_config.h

```diff
@@ -62,6 +62,12 @@ struct dynamic_config {
 
 	/* Specify the receiving timeouts until reporting an error. */
 	struct timeval   rcv_time_out;
+
+	/* The directory for Lua files of dynamic configuration. */
+	char             *lua_dy_base_dir;
```

Comment:
The line above and the lines around it are wrongly indented.

Reply:
It seems the indent is correct.

Reply:
It's showing right to me now. I guess it a GitHub bug.